src/coins.cpp

```diff
@@ -1277,6 +1289,14 @@ void SubtreeCache::PopSubtree(CCoinsView *parentView) {
     }
 }
 
+void SubtreeCache::ResetSubtrees() {
```

Comment:
It's an implementation detail that this method doesn't require the parent view, while other methods do.

I guess that's fine; if we were to refactor this then we'd remove the `parentView` argument from other methods (instead setting it on construction), rather than adding it to this method.